ktcacheorbust.cc

```diff
@@ -68,6 +83,18 @@ void CacheOrBust::configure(kt::TimedDB* dbary, size_t dbnum,
     }
     ++it;
   }
+
+  log(kt::ThreadedServer::Logger::SYSTEM, "CacheOrBust parameters:");
```

Comment:
in the spirit of DRY, is it possible to take `log(kt::ThreadedServer::Logger::SYSTEM, ...)` into a wrapped call?

something like:

```
void coblog(prefix, message)
{
    if strcmp(prefix, '') then { _prefix = 'CacheOrBust: ' } else { _prefix = prefix }
    log(kt::ThreadedServer::Logger::SYSTEM, printf("%s%s", prefix, msg)
}
```

Reply:
(fair warning: I presume that example as-is will be buggy, I typed that straight into the comment field...)

Reply:
I like the sentiment, but it's probably overkill here :)